packages/components/src/icon/index.tsx

```diff
@@ -1,50 +1,36 @@
 /**
- * External dependencies
+ * Forked from `@wordpress/components`
+ *
+ * Only forked for CSS collision safety, in case styles or classnames are added upstream.
+ *
+ * - Removed Gridicons support (non-critical).
```

Comment:
Worth keeping a simple log of changes like this?

Reply:
Let's try keeping track of these changes. We can re-evaluate later if it's too much effort / too low perceived value.

Do you think it's better to track them separately inside each component, or have a central log file?

Reply:
Maybe we can start with these inline docs, and when it becomes unwieldy we can move to a central log file.